PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -9,7 +9,7 @@
 local PlayersService = game:GetService('Players')
 local UserInputService = game:GetService('UserInputService')
 
-local HasVRAPI = pcall(function() local _ = UserInputService.VREnabled end)
+local HasVRAPI = pcall(function() return UserInputService.VREnabled and UserInputService.UserHeadCFrame end)
```

Comment:
Should update this to the new: UserInputService:GetUserCFrame(Enum.UserCFrame.Head) instead of using the deprecated one